guides/source/action_text_overview.md

```diff
@@ -242,4 +242,8 @@ Message.all.with_rich_text_content_and_embeds # Preload both body and attachment
     <action-text-attachment sgid="BAh7CEkiCG…"></action-text-attachment>
     ```
 
+## Internationalization (I18n)
+
+Action Text rich text can be translated using [Mobility Action Text](https://github.com/sedubois/mobility-actiontext).
```

Comment:
Not sure we should add this, there are many more tools and we can't be partial towards one library.

Reply:
Could you please share which other tools you are aware of, offering the same functionality?

The whole point of the library is to address a translation feature which is missing in ActionText.

Reply:
@vipulnsward